sdk/identity/identity-broker/test/internal/node/interactiveBrowserCredential.spec.ts

```diff
@@ -2,49 +2,42 @@
 // Licensed under the MIT License.
 import type { InteractiveBrowserCredentialNodeOptions } from "@azure/identity";
 import { InteractiveBrowserCredential, useIdentityPlugin } from "@azure/identity";
-import type { MsalTestCleanup } from "../../../../identity/test/node/msalNodeTestSetup";
-import { msalNodeTestSetup } from "../../../../identity/test/node/msalNodeTestSetup";
 import { PublicClientApplication } from "@azure/msal-node";
-import type Sinon from "sinon";
 import type { Recorder } from "@azure-tools/test-recorder";
 import { isLiveMode, env, isPlaybackMode } from "@azure-tools/test-recorder";
-import { nativeBrokerPlugin } from "../../../src";
+import { nativeBrokerPlugin } from "../../../src/index.js";
 import { isNodeLike } from "@azure/core-util";
-import { assert } from "@azure-tools/test-utils";
-import type http from "http";
+import type http from "node:http";
+import type { MockInstance } from "vitest";
+import { describe, it, assert, expect, vi, beforeEach, afterEach } from "vitest";
 
-describe("InteractiveBrowserCredential (internal)", function (this: Mocha.Suite) {
-  let cleanup: MsalTestCleanup;
+describe("InteractiveBrowserCredential (internal)", function () {
   let listen: http.Server | undefined;
-  let doGetTokenSpy: Sinon.SinonSpy;
+  let doGetTokenSpy: MockInstance;
   let recorder: Recorder;
 
-  beforeEach(async function (this: Mocha.Context) {
-    const setup = await msalNodeTestSetup(this.currentTest);
-    cleanup = setup.cleanup;
-    recorder = setup.recorder;
-
-    // getTokenSilentSpy = setup.sandbox.spy(MsalNode.prototype, "getTokenSilent");
-
-    doGetTokenSpy = setup.sandbox.spy(PublicClientApplication.prototype, "acquireTokenInteractive");
+  beforeEach(async function (ctx) {
+    doGetTokenSpy = vi.spyOn(PublicClientApplication.prototype, "acquireTokenInteractive");
   });
+
   afterEach(async function () {
     if (listen) {
       listen.close();
     }
 
-    await cleanup();
+    vi.restoreAllMocks();
   });
-  it("Throws error when no plugin is imported", async function (this: Mocha.Context) {
+
+  it("Throws error when no plugin is imported", async function (ctx) {
```

Comment:
Can we do `it.skipIf`?

Reply:
In this case bundling all these conditionals together results in harder to read code IMO. But if you feel strongly feel free to unresolve and let me know ! I can do that